org.zulip.Zulip.json

```diff
@@ -0,0 +1,47 @@
+{
+  "app-id": "org.zulip.Zulip",
+  "branch": "stable",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "zulip",
+  "rename-icon": "zulip",
+  "rename-desktop-file": "zulip.desktop",
+  "separate-locales": false,
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=home",
+    "--share=ipc",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "zulip",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar -x Zulip-2.0.0-amd64.deb",
+        "tar -xf data.tar.xz",
+        "cp -r usr/* /app",
+        "mkdir -p /app/bin",
+        "cp -r opt/Zulip/* /app/bin",
+        "desktop-file-edit --set-key=Exec --set-value='zulip %U' /app/share/applications/zulip.desktop",
+        "install -Dm644 org.zulip.Zulip.appdata.xml /app/share/appdata/org.zulip.Zulip.appdata.xml"
+      ],
+      "sources": [
+        {
```

Comment:
Please add source tarball as well so it's included into Sources extension.

Reply:
Done